drivers/tee/optee/core.c

```diff
@@ -219,7 +219,7 @@ static void optee_get_version(struct tee_device *teedev,
 	struct tee_ioctl_version_data v = {
 		.impl_id = TEE_IMPL_ID_OPTEE,
 		.impl_caps = TEE_OPTEE_CAP_TZ,
-		.gen_caps = TEE_GEN_CAP_GP,
+		.gen_caps = (TEE_GEN_CAP_GP | TEE_GEN_CAP_MEMREF_NULL),
```

Comment:
Since there's a corresponding ABI change to secure world I think we should enable this only if `OPTEE_SMC_SEC_CAP_MEMREF_NULL` is received from secure world.

Reply:
Done